codespan-reporting/tests/snapshots/term__same_ranges__rich_no_color.snap

```diff
@@ -6,7 +6,9 @@ error: Unexpected token
   ┌─ same_range:1:5
   │
 1 │ ::S { }
-  │     ^ Unexpected '{'
-  │     - Expected '('
+  │     ^
+  │     │
+  │     Unexpected '{'
+  │     Expected '('
```

Comment:
This should ultimately address the main complaint that was described in #231 by @tnowacki. cc. https://github.com/libra/libra/pull/3447